### ISSUE TITLE
About Loop and HardwareAcceleration Properties

### ISSUE THREAD

**Issue by SailorTurkey** [martijn00/LottieXamarin#281]
Hello, 
After updating nuget package, i saw some properties are removed.
Readme examples still use Loop="true".

i replaced Loop="True" to RepeatMode="Infinite", and from checking the code i guess animationview will use HW Acceleration wherever possible.

Is this the right approach?
Thanks

**Comment by petro2050** [comment]
RepeatMode="Infinite" is not working for me

**Comment by sereoja** [comment]
I do have same issue with last update 4.0.0 regarding Loop parameter missing. Please return it.

**Comment by Abesoddy** [comment]
Same issue... I remain in previous version while waiting for an answer ...

**Comment by petro2050** [comment]
I think they removed the Loop parameter on purpose in place of RepeatMode, but RepeatMode does not work. On Android, you could set RepeatCount to int.MaxValue to simulate looping, but on iOS RepeatCount doesn't work. So, looping is broken with the latest release.

**Comment by martijn00** [comment]
Try RepeatMode in 4.0.1

**Comment by sereoja** [comment]
Please see my comments below.
These observations are made after 4.0.1 Update on Android Emulator.

Infinite Mode. 
Expectation: Repeat the animation indefinitely.   
Actually: Works like Reverse Mode. without RepeatCount it runs only once, but if you set RepeatCount to a number it would run animation back and forth to what Reverse Mode would be expected to.

Restart Mode.
Works as expected.

Reverse Mode.
Expectation: When the animation reaches the end and RepeatCount is Infinite or a positive value, the animation reverses direction on every iteration.
Observation: The animation acts same as Restart Mode, no reverse direction is observed.

I would like to encourage you and let you know that you've done a great job with this plugin and I personally greatly appreciate your efforts to provide and improve this plugin for the community. Great job.

And lastly, one more thing. please update readme file, since it still shows to use Loop parameter.
<img width="1227" alt="Screen Shot 2020-09-28 at 8 53 42 AM" src="https://user-images.githubusercontent.com/19211452/94456809-9603f380-0168-11eb-8cae-7ba001317c04.png">

**Comment by martijn00** [comment]
@sereoja Please open a new issue with the issues.
